sdk/eventhub/Microsoft.Azure.Management.EventHub/CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+2.5.0
```

Comment:
This proj is management plane. I suppose it's an accident?

`sdk/eventhub/Microsoft.Azure.Management.EventHub/src/Microsoft.Azure.Management.EventHub.csproj` too

Reply:
+1